src/views/Viewer.vue

```diff
@@ -694,7 +699,7 @@ export default {
 			this.handlerId = handler.id
 
 			// check if part of a group, if so retrieve full files list
-			const group = this.mimeGroups[mime]
+			const group = this.mimeGroups[mime] ?? this.mimeGroups['*/*']
```

Comment:
* let's split this part like in https://github.com/IONOS-Productivity/nc-viewer/tree/review/fb/feat/add_default_previewer/default_component